pkg/client/unversioned/request_test.go

```diff
@@ -719,6 +723,42 @@ func TestDoRequestNewWay(t *testing.T) {
 	fakeHandler.ValidateRequest(t, requestURL, "POST", &reqBody)
 }
 
+// This test assumes that the client implementation backs off exponentially, for an individual request.
+func TestBackoffLifecycle(t *testing.T) {
+	count := 0
+	testServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		count++
+		t.Logf("Attempt %d", count)
+		if count == 5 || count == 9 {
+			w.WriteHeader(http.StatusOK)
+			return
+		} else {
+			w.WriteHeader(http.StatusGatewayTimeout)
+			return
+		}
+	}))
+	defer testServer.Close()
+	c := testRESTClient(t, testServer)
+
+	// Test backoff recovery and increase.  This correlates to the constants
+	// which are used in the server implementation returning StatusOK above.
+	seconds := []int{0, 1, 2, 4, 8, 0, 1, 2, 4, 0}
+	request := c.Verb("POST").Prefix("backofftest").Suffix("abc")
+	request.backoffMgr = &URLBackoff{
```

Comment:
hi @krousey as per your suggestion,  this is now fixed to do a direct set of the backoff policy rather than env vars.